healthcare/api-client/v1/fhir/README.rst

```diff
@@ -100,22 +100,49 @@ To run this sample:
     positional arguments:
       {create-fhir-store,delete-fhir-store,get-fhir-store,get-fhir-store-metadata,list-fhir-stores,patch-fhir-store,import-fhir-resources,export-fhir-store-gcs,get_iam_policy,set_iam_policy}
         create-fhir-store   Creates a new FHIR store within the parent dataset.
-        delete-fhir-store   Deletes the specified FHIR store.
-        get-fhir-store      Gets the specified FHIR store.
+                            See https://github.com/GoogleCloudPlatform/python-
+                            docs-samples/tree/master/healthcare/api-client/v1/fhir
+                            before running the sample.
```

Comment:
Re: `See https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/healthcare/api-client/v1/fhir before running the sample.`

Does it make sense to add this sentence at the beginning of the README (or anywhere else you think appropriate)? It bulks up the documentation a lot and lost the brevity.

Not blocking though.

Reply:
I did this following the "Descriptive to-level comment" in our rubric. Do you think it's best to just have it once at the top of the Python file? I wanted it to be visible to people copy/pasting snippets from the docs.

Reply:
Ok, it sounds good if you have a reason.

Reply:
Actually looking at it now it does make sense to manually remove that sentence from each method in README.rst. We'll just have to do the manual removal any time we auto-gen README.rst.